bin/GuiConfigs/YoRHa by Ani.qss

```diff
@@ -243,6 +243,12 @@ QSpinBox {
 	border-radius: 0.10em;
 }
 
+/* QDoubleSpinBox (Pads -> Mouse Acceleration -> x/y) */
+QDoubleSpinBox {
```

Comment:
Probably better to add it together with QSpinBox since it's the same styling

So you only need to change line 239 to have
`QSpinBox, QDoubleSpinBox {`

And then add the QDoubleSpinBox comment after the QSpinBox comment on line 238

Reply:
Done. Also, I noticed that some other themes also don't style `QDoubleSpinBox`. Should I broaden the scope of pull request and update them?

Reply:
Sure